### ISSUE TITLE
Issue with aurelia/binding typings 

### ISSUE THREAD

**Issue by dhanapalschandran** [aurelia/binding#736]
![image 6](https://user-images.githubusercontent.com/9696305/52210338-c8cee900-28ac-11e9-9d24-eb5802105b15.png)
Hi Team,

  After the recent commit we are getting the build issue related to Disposable, Callable interface in typings, which was expected in aurelia templating, 
 Now once we fall back to 2.1.7 version, it works fine.
 Kindly let us know when it will be resolved.

**Comment by bigopon** [comment]
@dhanapalschandran Thanks for filing this. Can you help paste the full error message? It seems important parts of the error were not captured.

**Comment by dhanapalschandran** [comment]
> @dhanapalschandran Thanks for filing this. Can you help paste the full error message? It seems important parts of the error were not captured.

Hi @bigopon , I have provided the screenshot of the error we got during gulp build, Thanks.

**Comment by bigopon** [comment]
@dhanapalschandran I see the updated picture. I'm curious about your usage, as it seems you are importing `bindingMode` from `aurelia-templating`?
As far as I know, it should have never worked because it's the import from `aurelia-binding`. The only module that reexport `bindingMode` is `aurelia-framework`, which means:

```diff
// not working
- import { bindingMode } from 'aurelia-templating';
// working
+ import { bindingMode } from 'aurelia-framework';
```

**Comment by dhanapalschandran** [comment]
> @dhanapalschandran I see the updated picture. I'm curious about your usage, as it seems you are importing `bindingMode` from `aurelia-templating`?
> As far as I know, it should have never worked because it's the import from `aurelia-binding`. The only module that reexport `bindingMode` is `aurelia-framework`, which means:
> 
> ```diff
> // not working
> - import { bindingMode } from 'aurelia-templating';
> // working
> + import { bindingMode } from 'aurelia-framework';
> ```

Yes correct, this was error was popped from aurelia-templating index.ts

**Comment by fkleuver** [comment]
@dhanapalschandran Are you somehow directly installing from the master branch? We are moving towards a new, more streamlined workflow with CircleCI but it seems the build process is not entirely correct yet. The dist files do seem to be missing stuff. Will fix this asap

**Comment by jmezach** [comment]
I think that d359e83 might have broken this. For example, with aurelia-binding 2.2.0 I can import the disableConnectQueue function from aurelia-binding, but with 2.2.1 I cannot, even though it's available at runtime. This seems like a regression.

**Comment by bigopon** [comment]
@jmezach nice, thanks for the catch. The issue comes from the fact that typings was commited to `dist` file in this https://github.com/aurelia/binding/commit/f4012aa47b450ee0bb13c87518d31a695767b75d while it should have been committed to typings file in `src`. I'll fix that soon.

**Comment by bigopon** [comment]
All issues have been fixed after #746 .
It can be closed @EisenbergEffect @dhanapalschandran
